gothemis/session/session_test.go

```diff
@@ -223,6 +223,50 @@ func testSession(keytype int, t *testing.T) {
 	<-finCh
 }
 
+func TestValidKeyHandling(t *testing.T) {
+	kpa, err := keys.New(keys.TypeEC)
+	if nil != err {
```

Comment:
common way to check errors is `if err != nil` in golang. It's okay do such checks in C to avoid errors like `if err = nil`. But in golang we cannot assign value in `if` statements

Reply:
`if nil != err` was just a copypase, I would usually write `if err != nil` as well as `if a == 300` instead of `if 300 == a` which IMO looks kinda ugly.

BTW, not too long ago Python introduced `:=` operator just to use it in `if`s and loops

Reply:
yes, but gothemis in most cases will be read by golang developers, not python)